src/firefox/index.ts

```diff
@@ -143,6 +145,26 @@ export default class {
     }
   }
 
+  getKillCmd(pid: number) {
+    return global.platform.win32 ? `taskkill /F /PID ${pid}` : `kill ${pid}`
+  }
+
+  async close() {
+    const processes: Process[] = await find('name', /firefox/i)
```

Comment:
This should be good enough to find processes in Linux and MacOs. The package claims to support Windows as well, so I hope it works, but haven't tested it.

Reply:
Wouldn't this potentially find another Firefox instance (a Firefox launched by the user; not our Point Browser)?

We should find a way to store the PID of the Firefox instance that we launch with the dashboard.

Reply:
Yes, it will find all Firefox instances, not just Point Browser, but then we filter out the Point Browser instances by looking for the string "point-browser" in the process `cmd`.

The only reason I didn't try to store the PID when launching Firefox, is that I see two running processes in Linux and killing one does not kill the other, even though one is the parent. So I thought it would be better to search for all firefox processes and filter the ones we need to kill. But I can look into the launching method and see what PID we get there.